frontend/public/components/search-filter-dropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem, TextInput } from '@patternfly/react-core';
+import { CaretDownIcon, FilterIcon } from '@patternfly/react-icons';
+
+export const SearchFilterDropdown: React.SFC<SearchFilterDropdownProps> = (props) => {
+  const [isOpen, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState('Label');
+  const [inputValue, setInputValue] = React.useState('');
+
+  const { onChange } = props;
+
+  const onToggle = (open: boolean) => setOpen(open);
+  const onSelect = (event: React.SyntheticEvent) => {
+    setSelected((event.target as HTMLInputElement).name);
+    setOpen(!isOpen);
+  };
+  const dropdownItems = [
+    <DropdownItem key="label-action" name="Label" component="button">
+      Label
+    </DropdownItem>,
+    <DropdownItem key="name-action" name="Name" component="button">
+      Name
+    </DropdownItem>,
+  ];
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      onChange(selected.toLowerCase(), inputValue);
+      setInputValue('');
+    }
+  };
+
+  return (
+    <div className="form-group co-search-group__filter">
+      <div className="pf-c-input-group">
+        <Dropdown
+          onSelect={onSelect}
+          toggle={
+            <DropdownToggle id="toggle-id" onToggle={onToggle} iconComponent={CaretDownIcon}>
+              <>
+                <FilterIcon className="span--icon__right-margin" /> {selected}
+              </>
+            </DropdownToggle>
+          }
+          isOpen={isOpen}
+          dropdownItems={dropdownItems}
+        />
+        <TextInput
+          onChange={setInputValue}
+          placeholder={selected === 'Label' ? 'Filter by label...' : 'Filter by name...'}
```

Comment:
We can't just use a plain text input for labels. We should use the `SelectorInput` component for labels

Reply:
That's the PF TextInput.  So, the SelectorInput creates the tags that are in badges on the input field but I place those items in the new toolbar.  I need to keep this using the TextInput.